docs/graphql-api.md

```diff
@@ -0,0 +1,184 @@
+---
+title: GraphQL API
+keywords: homepage
+sidebar: talk_sidebar
+permalink: /graphql-api.html
+summary:
+---
+
+## GraphiQL
+
+Explore the GraphQL API at
+
+http://localhost:3000/api/v1/graph/iql
+
+with the “Documentation Explorer” on the right side.
+
+
+![GraphiQL screenshot](https://user-images.githubusercontent.com/815894/28073533-47debff6-6656-11e7-82fb-eb5aebefa350.png)
+
+
+
+
+
+After logged-in as admin at admin/configure you are able to use GrapiQL to show admin/user data.
+
+
+
+## First sample query
```

Comment:
If the samples and the images/screenshots are removed from this page, there is not much left.

Would it then be better to add the direction to http://graphql.org/learn/ on this http://localhost:4000/tools.html page and remove docs/graphql-api.md?